src/ceph-detect-init/requirements.txt

```diff
@@ -1 +0,0 @@
-argparse
```

Comment:
Removing it since builtin means this tool can be built without it trying to hit network at all (it ignored the system copy otherwise)